src/makeRobot.js

```diff
@@ -38,7 +38,63 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const Robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    goForward(valY = 1) {
```

Comment:
```suggestion
    goForward(step = 1) {
```
По суті, ми сюди передаємо крок, з яким потрібно рухатись. Тому така назва підійде більше